modules/standard/Math.chpl

```diff
@@ -236,7 +236,8 @@ module Math {
     - ``proc asin(imag): imag``
     - ``proc atanh(imag): imag``
   */
-  config param useNewImaginaryTrig = false;
+  @deprecated(notes="'newSliceRule' is deprecated and will be removed in a future release. It now has no effect, and is 'true' by default.")
```

Comment:
typo: `newSliceRule` -> `useNewImaginaryTrig`

Reply:
also, consider updating the docstring/marking this as nodoc

Reply:
🤦